Hardware requirement and coding style (#77)
OpenSILEX/docs-community-dev: docs/localInstallation.md

## 1. Prerequisite

### Hardware

> We strongly recommend a server or a virtual machines on a cloud compute with :

Reviewer: ```suggestion
We strongly recommend a server or virtual machines on a compute cloud with :
```

- Processor: recommended 4 CPU 2.30GHz
- Ethernet connection

> - Hard Drive: minimum 500 Go, depend of you data.

Reviewer: ```suggestion
- Hard Drive: minimum 500 Go, depending on your data volume.
```
